src/linear_solvers/direct.jl

```diff
@@ -24,16 +24,31 @@ function scs_init(
     )::Ptr{Cvoid}
 end
 
+function scs_update(
+    ::Type{DirectSolver},
+    work::Ptr{Cvoid},
+    b::Vector{Float64},
+    c::Vector{Float64},
+)
+    return @ccall direct.scs_update(
+        work::Ptr{Cvoid},
+        b::Ref{Float64},
+        c::Ref{Float64},
+    )::Clonglong
+end
+
 function scs_solve(
     ::Type{DirectSolver},
     work::Ptr{Cvoid},
     solution::ScsSolution,
     info::ScsInfo{I},
+    warm_start::Integer,
```

Comment:
:( More breaking changes

Reply:
this seems to be breaking, but we can fix it by giving it the default value `0`.

Reply:
Also: we do export `scs_solve`, but this particular method isn't even documented (e.g. because of its arguments which are private), so I think these changes justify only minor version bump (but maybe we should opt for `1.1.0`?

Reply:
Again, I meant in the underlying library. I think our only public API is the MOI optimizer and the full `SCS.scs_solve`.